modules/test_user_dashboard/app/models/test_user_dashboard/tud_account.rb

```diff
@@ -9,11 +9,41 @@ class TudAccount < ApplicationRecord
     validates :first_name, :last_name, :email, :gender, presence: true
     validates :email, uniqueness: true
 
+    serialize :services
+
     # uncomment when adding id_provider column information
     # validates :id_provider, presence: true
     # validates :id_provider, inclusion: { in: ID_PROVIDERS }
     def available?
       checkout_time.nil?
     end
+
+    def user_identity
+      unless (identity = UserIdentity.find(account_uuid))
+        identity = UserIdentity.create(uuid: account_uuid,
+                                       email: email,
+                                       first_name: first_name,
+                                       last_name: last_name,
+                                       gender: gender,
+                                       birth_date: birth_date.to_s(:iso_8601),
+                                       ssn: ssn,
+                                       loa: { lowest: 1, highest: 3 })
```

Comment:
As a fellow writer of beautiful code, you can do this and rubocop will be happy:

```suggestion
        identity = UserIdentity.create(
          uuid: account_uuid,
          email: email,
          first_name: first_name,
          last_name: last_name,
          gender: gender,
          birth_date: birth_date.to_s(:iso_8601),
          ssn: ssn,
          loa: { lowest: 1, highest: 3 }
        )
```